build/helper/metadata_add_all.py

```diff
@@ -121,6 +121,19 @@ def _add_library_call_name(parameter, session_name):
     parameter['library_call_name'] = library_call_name
 
 
+def _add_default_value_name(parameter):
```

Comment:
IMO: Concatenating two strings in metadata probably doesn't warrant inserting into metadata.

Reply:
If you disagree with this, you can merge. PR accepted.

Reply:
The advantage of adding this to metadata is that the render portion of get_params_snippet() has no name specific code. Right now it is three lines:
``` python
for x in params_to_use:
    snippets.append(x[name_to_use])
return ', '.join(snippets)
```

If this isn't added to metadata, then there would need to be something like this:
``` python
for x in params_to_use:
    if name_to_use == 'python_name_with_default':
        <do the concatenation here>
    else:
        snippets.append(x[name_to_use])
return ', '.join(snippets)
```
This would make that function even longer, which I was trying to avoid.

Reply:
Well, that function needs to be broken up / refactored.